fuzz_test.go

```diff
@@ -105,3 +105,40 @@ func TestCrashNSEC(t *testing.T) {
 		t.Fatalf("expected length of %d, got %d", expectedLength, l)
 	}
 }
+
+// TestCrashNSEC3 tests generated using fuzz.go and with a message pack
+// containing the following bytes:
+// "0000\x00\x00000000\x00\x00200000" +
+// "0\x00\v0000\x00\x00#\x0300\x00\x00\x00\x1a000" +
+// "000\x00\v00\x0200\x00\x03000\x00"
+// That byte sequence, when Unpack() and subsequential Pack() created a
+// panic: runtime error: slice bounds out of range
+// which was attributed to the fact that NSEC RR length computation was different (and smaller)
+// then when within packDataNsec.
+func TestCrashNSEC3(t *testing.T) {
```

Comment:
Isn't this a dup of the NSEC test then?

Reply:
so it is a similar issue, but with a different input. I do admit that I copied the comment from the NSEC test and adapted it, but forgot to rename NSEC to NSEC3 in 1 spot.